Remotion/ObjectBinding/Web.ClientScript/Scripts/typings/Utilities.d.ts

```diff
@@ -75,3 +75,11 @@ declare class LayoutUtility {
     static Show(element: HTMLElement): void;
     static FormatPixelProperty(value: string | number): string
 }
+/**
```

Comment:
```suggestion
}

/**
```

Reply:
This file is auto-generated so I would not adjust formatting. This file is also just a "temporary" fix until we fixed the typescript project references between web and objectbinding.